packages/block-library/src/avatar/editor.scss

```
@@ -1,3 +1,9 @@
 .wp-block-avatar__image img {
 	width: 100%;
 }
+
+.aligncenter {
+	.components-resizable-box__container {
+		margin: 0 auto;
+	}
+}
```

This may affect other components apart from Avatar. Maybe we could be more class specific.

```suggestion
.wp-block-avatar.aligncenter > .components-resizable-box__container {
	margin: 0 auto;
}
```